### PR TITLE
Added the cut thief voice lines.

### LINKED ISSUE #17210: Include the newly discovered RA beta Thief VO lines.
https://www.reddit.com/r/commandandconquer/comments/dflqti/a_present_to_the_cc_community/

### PR DESCRIPTION
**Pull request by Leo512bit**
But how are implement this for the remaster?

### CODE REVIEW

File: mods/ra/audio/voices.yaml [hunk 1]
```diff
@@ -86,7 +86,7 @@ SpyVoice:
 ThiefVoice:
 	Voices:
 		Select: swhat1,syeah1
-		Action: saffirm1,smout1,sokay1
+		Action: saffirm1,smout1,sokay1,scandy,sclean1,ssteal1
```
Comment (Mailaender) [inline mods/ra/audio/voices.yaml:89]:
```suggestion
		Move: saffirm1,smout1,sokay1
		Action: scandy,sclean1,ssteal1
```
like the `Spy` above.

Comment (abcdefg30) [reply mods/ra/audio/voices.yaml:89]: That the `Mobile` trait has to change to use the `Move` voice as well, I suppose? (Not a problem, just something to set up.)

**Review by Mailaender — DISMISSED** [review]

File: mods/ra/audio/voices.yaml (hunk 1 above)
Comment (Mailaender) [reply mods/ra/audio/voices.yaml:89]: https://github.com/OpenRA/OpenRA/blob/e685731b337d76ba4d03b7dc84105029870236d4/mods/ra/rules/infantry.yaml#L652 still needs `Voice: Move`

Comment (Leo512bit) [reply mods/ra/audio/voices.yaml:89]: Done.

**Comment by Leo512bit** [comment]
What changes are needed?

**Comment by Mailaender** [comment]
You need to https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/managing-commits/squashing-commits

**Comment by Leo512bit** [comment]
Can't squash it for some reason I'll open up a new PR

**Comment by Smittytron** [comment]
> Can't squash it for some reason I'll open up a new PR

You can always ask us in the OpenRA Discord if you run into any git troubles.

**Comment by Leo512bit** [comment]
I don't use discord nor do I want to.

**Comment by Mailaender** [comment]
There is also https://web.libera.chat/#openra as an alternative, though not as populated.

**Pull request closed without merging**